### ISSUE TITLE
Add additional configuration parameters

### ISSUE THREAD

**Issue by mguissine** [pentestfail/TA-Github#7]
Hi Evan,

Just downloaded your add-on and it works great. Would be nice though if you add an ability to provide additional parameters in the inputs configurations. In my case, I need to pull only `closed` issues into the splunk however github's default api pulls `open` issues only.

To work around, I had to change `input_module_github_api_repos_issues` and to add ` parameter['state'] = "closed"`, see below

```
    # Create API request parameters    
    auth = base64.b64encode(git_username + ":" + git_password).decode("ascii")
    header =  {'Authorization': 'Basic {}'.format(auth)}
    parameter = {}
    parameter['since'] = last_status
    parameter['per_page'] = git_pagesize
    parameter['state'] = "closed"
    method = 'GET'
```

**Comment by pentestfail** [comment]
Agreed, I've been waiting on Splunk to update some of their guidance and developer tooling to address recent changes to requirements in AppInspect and Splunkbase.  Sadly they didn't release anything except for Splunk Cloud related changes at .CONF (at least not that I've seen publicly) so I guess I'll add this to the backlog of updating the app for v7.x Bootstrap UI and CSS changes once I figure them all out.
